src/NearestNeighbors.jl

```diff
@@ -1,11 +1,12 @@
-__precompile__()
+# __precompile__()
```

Comment:
StaticArrays should now be precompiled, if that was the problem here (use 0.0.4 in the REQUIRE file above)